curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646bb08a93cf75070ca0e505.md

```diff
@@ -0,0 +1,141 @@
+---
+id: 646bb08a93cf75070ca0e505
+title: Step 27
+challengeType: 0
+dashedName: step-27
+---
+
+# --description--
+
+Target the id selector named `#current-date` in the CSS file and add the following properties:
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed